medical_compliance/medical_compliance/api/tasks.py

```diff
@@ -12,58 +12,35 @@
 from kombu import Queue, Exchange
 from django.conf import settings #noqa
 
-from models import WithingsMeasurement
-from withings import WithingsCredentials, WithingsApi, WithingsMeasures
+from models import WeightMeasurement
+from analyzers.weight_analyzers import analyze_weights
 
-logger = get_task_logger("medical_compliance.fetch_measurement")
+logger = get_task_logger("medical_compliance_measurements.fetch_weight_measurement")
 
 
 app = Celery('api.tasks', broker=settings.BROKER_URL)
 app.conf.update(
-    CELERY_DEFAULT_QUEUE='withings_measurements',
+    CELERY_DEFAULT_QUEUE='medical_compliance_measurements',
     CELERY_QUEUES=(
-        Queue('withings_measurements', Exchange('withings_measurements'), routing_key='withings_measurements'),
+        Queue('medical_compliance_measurements', Exchange('medical_compliance_measurements'), routing_key='medical_compliance_measurements'),
     ),
 )
 
 
-@app.task(name='medical_compliance.fetch_measurement')
-def fetch_measurement(userid, start_ts, end_ts, measurement_type_id):
+@app.task(name='medical_compliance_measurements.fetch_weight_measurement')
+def fetch_weight_measurement(user_id, input_source, measurement_unit, timestamp, timezone, value):
```

Comment:
@paraschivionut I find the method name confusing - it's for saving a measurement, not for fetching it :)